docs/components/configMaps.md

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.configMaps`
+
+This grade provides the configuration maps for the gamepad in the form of component's model data.
+
+## Using this grade
+
+This can be used by passing it to the component's `gradeNames` option.
+
+``` javascript
+fluid.defaults("my.configMaps.grade", {
+    gradeNames: ["gamepad.configMaps"]
+};
+
+var configMapsInstance = my.configMaps.grade();
+```
+
+## Component Options
+
+This grade doesn't use or provide any configuration options. It only uses the model data to provide gamepad configuration
+maps to other components. However, the grade can be extended to provide custom configuration options.
+
+## Model Structure
+
+The grade stores the model data in the following format:
+
+``` javascript
+model: {
+    map: {
+        buttons: {
+            "0": {
+                defaultAction: "previousPageInHistory",
+                currentAction: null,
+                speedFactor: 1,
+                background: false
+            },
+            // The buttons below use same format as above but have different values for configuration options (keys).
+            "1": {...},
+            "2": {...},
+            "3": {...},
+            "4": {...},
+            "5": {...},
+            "6": {...},
+            "7": {...},
+            "8": {...},
+            "9": {...},
+            "10": {...},
+            "11": {...},
+            "12": {...},
+            "13": {...},
+            "14": {...},
+            "15": {...},
+            // The buttons below are reserved for special functionality so they cannot be configured.
```

Comment:
You don't have to write it out in the docs, but I'm curious what you have in mind.  I'm also curious if you have anything that has more than 16 buttons.

Reply:
The `Button 16` is the "badge icon," which we were planning to use for opening the configuration panel if the Chrome extensions allow us to do so (as we discussed in one of our email conversations). And the `Button 17` is the touchpad which acts as a mouse in some operating systems (at least in Ubuntu), so I isolated these buttons. But should I remove this part (comment and model variables) from the docs or keep it as it currently is?

Reply:
Thanks, somehow I had the idea that 15 was the badge, and I haven't hooked up my PS4 Dualshock in a while, so I forgot about the touchpad.  I think it's fine to hold on to it for special features.